contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -282,6 +285,7 @@ GetBulkLoadRequest(StringInfo message)
 			{
 				uint16_t tableLen = 0;
 				CheckMessageHasEnoughBytesToRead(&message, sizeof(uint32_t));
+				Assert(sizeof(uint32_t)<=sizeof(colmetadata[currentColumn].maxLen));
```

Comment:
This again seems redundant since its sizeof(uint32_t)<=sizeof(uint32_t) which is always going to be true

Reply:
Will remove it.